lib/rubocop/cop/style/redundant_self.rb

```diff
@@ -92,13 +90,11 @@ def on_blockarg(node)
         end
 
         def on_masgn(node)
-          lhs, rhs = *node
-          add_masgn_lhs_variables(rhs, lhs)
+          add_masgn_lhs_variables(node.expression, node.lhs)
```

Comment:
You could use `node.rhs` for balance if you'd like (I added it as an alias with this in mind but up to you of course!)

Reply:
Good idea, but… I can use `#rhs` here, but not in line 97. Do you think we need an `alias rhs expression` for `AsgnNode` too?

Reply:
I think it's ok not having `rhs` for your specific example because you're also not using `lhs`.

That being said, it _could_ help allow `masgn` to be treated similiarly to the other assignment nodes, or at least have a common nomenclature for assignments. If we were to do so we should add `lhs` too (I guess an alias for `name`). The obvious caveat there is that for a generic assignment `lhs` is either a string or a `mlhs`.

Reply:
> I'm happy to open a rubocop-ast PR tomorrow if you don't 

I will leave that to you 😄

Reply:
Sounds good, I opened rubocop/rubocop-ast#326.